Enterprise-CMCS/eAPD
# Tbolt/4371 mmis nav updates

Resolves #4371 

### Description
Updates side nav to support different apdTypes: HITECH and MMIS

### Significant changes or possible side effects
* Navigation is updated to reflect different sections based on whether the apd is HITECH or MMIS type

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| viewing an apd | its a MMIS apd | should show 4 new sections in the nav | jest |
| viewing an apd | its a HITECH apd | should not show 4 new sections in the nav | jest |

### Steps to manually verify this change
1. Select an APD with HITECH type (existing seed)
2. Verify the side nav is according to the original ticket / figma / google doc
3. Select and APD with MMIS type (existing seed)
4. Verify the side nav is according to the original ticket / figma / google doc

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
